### ISSUE TITLE
Change speed without pressing the stop button (and reread everything)

### ISSUE THREAD

**Issue by 0LMA** [rNeomy/reader-view#160]
Hello!

Unfortunately you cannot (or at least I cannot) change the speed of text-to-speech without then pressing the stop button and restart from the beginning of the text. If you change the control, it does not affect the reading speed until you do that.

Thank you

**Comment by rNeomy** [comment]
True, any changes made to the rate property will only take effect on subsequent calls to the `SpeechSynthesis.speak()` method, and will not affect any speech that is currently being played. This is by `SpeechSynthesis` design.
